.github/workflows/deploy.yml

```diff
@@ -191,3 +193,102 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           AWS_DEFAULT_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
+
+  update-apollo-studio-graphs:
```

Comment:
idea: to make this quicker we could start and create the *.graphql files as initial step of the deploy

and after terraform just publish them

something similar to what we do for the emails